student/pom.xml

```diff
@@ -25,7 +25,7 @@
 		<progman-client.version>3.0.1</progman-client.version>
 		<sb11-mna-client.version>3.0.0</sb11-mna-client.version>
 		<sb11-shared-code.version>3.0.4</sb11-shared-code.version>
-		<tds-exam-client.version>0.0.12</tds-exam-client.version>
+		<tds-exam-client.version>0.0.13-SNAPSHOT</tds-exam-client.version>
```

Comment:
I will clean up this file and point to latest versions once things are released and before merging.